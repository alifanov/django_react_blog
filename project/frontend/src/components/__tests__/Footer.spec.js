import React from "react";
import Footer from "../footer";
import { shallow } from "../../enzyme";

describe(">>> Footer test", () => {
  let footer;
  beforeEach(() => {
    footer = shallow(<Footer />);
  });

  it("+++ render Footer component", () => {
    expect(footer.length).toEqual(1);
  });
  it("+++ check text for Footer", () => {
    expect(
      footer
        .find("div>p")
        .at(1)
        .text()
    ).toEqual("Blog example");
  });
});
