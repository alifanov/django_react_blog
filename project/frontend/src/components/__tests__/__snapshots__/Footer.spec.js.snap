// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>Footer --- Snapshot +++capturing Snapshot of Footer 1`] = `
<footer
  className="text-muted"
>
  <div
    className="container"
  >
    <p
      className="float-right"
    >
      <a
        href="#"
      >
        Back to top
      </a>
    </p>
    <p>
      Blog example
    </p>
  </div>
</footer>
`;
